eng/CodeAnalysis.ruleset

```diff
@@ -310,35 +310,35 @@
     <Rule Id="SA1141" Action="Warning" />       <!-- Use tuple syntax -->
     <Rule Id="SA1142" Action="Warning" />       <!-- Refer to tuple elements by name -->
     <Rule Id="SA1200" Action="None" />          <!-- Using directive should appear within a namespace declaration -->
-    <Rule Id="SA1201" Action="None" />          <!-- Elements should appear in the correct order -->
-    <Rule Id="SA1202" Action="None" />          <!-- Elements should be ordered by access -->
-    <Rule Id="SA1203" Action="None" />          <!-- Constants should appear before fields -->
-    <Rule Id="SA1204" Action="None" />          <!-- Static elements should appear before instance elements -->
+    <Rule Id="SA1201" Action="Info" TemporarilyDisabled="true" />          <!-- Elements should appear in the correct order -->
```

Comment:
Is this about alphabetic order? 
If so, I'm not sure about it for some cases - for examples for tests I sometimes prefer them to be logically ordered - happy path goes first, edge cases last.

Reply:
No, this is about how fields should come before properties and methods. More details here: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1201.md